criu/mount.c

```diff
@@ -2472,6 +2477,14 @@ static int do_close_one(struct mount_info *mi)
 	return 0;
 }
 
+static int set_unbindable(struct mount_info *mi)
+{
+	if (mi->flags & MS_UNBINDABLE && !mnt_is_overmounted(mi)) {
```

Comment:
if ((mi->flags & MS_UNBINDABLE) && !mnt_is_overmounted(mi))

Reply:
I mentioned that you ask me to change such conditions for several times already. Do we have some code-style rule for it? Why operator precedence is not enough?

Reply:
No, we don't. For me, this form looks more readable. I found that your form is widely used in the Linux kernel sources, so I think you can leave it as is.

Reply:
Lets be lazy and leave it as in latest push - with parentheses.